src/shared/ui/common/regionSubmenu.ts

```diff
@@ -46,7 +46,7 @@ export class RegionSubmenu<T> extends Prompter<RegionSubmenuResponse<T>> {
                 data: undefined,
             },
             {
-                label: 'Switch region',
+                label: 'Switch Regions',
```

Comment:
kind of strange to make it plural?

Reply:
originally suggested by Jo, but double checking about the plural piece.